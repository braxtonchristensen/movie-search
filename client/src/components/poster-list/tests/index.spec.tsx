import React from 'react';
import renderer from 'react-test-renderer';
import PosterList from '../index';
import Poster, { PosterSize } from '../../poster';

describe('<PosterList />', () => {
  describe('snapshots', () => {
    it('should match snapshot', () => {
      const rendered = renderer.create(
        <PosterList>
          <Poster
            size={ PosterSize.S }
            path="/gpxjoE0yvRwIhFEJgNArtKtaN7S.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/4RnCeRzvI1xk5tuNWjpDKzSnJDk.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/3PEAkZHa8ehfUkuKbzmQNRTTAAs.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/zDJ4GVch7Iw2i0j84ceoElRUZUR.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/gpxjoE0yvRwIhFEJgNArtKtaN7S.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/4RnCeRzvI1xk5tuNWjpDKzSnJDk.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/3PEAkZHa8ehfUkuKbzmQNRTTAAs.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
          <Poster
            size={ PosterSize.S }
            path="/zDJ4GVch7Iw2i0j84ceoElRUZUR.jpg"
            title="This will be dynamic and will be used for accessibility"
          />
        </PosterList>
      ).toJSON();

      expect(rendered).toMatchSnapshot();
    });
  });
});
