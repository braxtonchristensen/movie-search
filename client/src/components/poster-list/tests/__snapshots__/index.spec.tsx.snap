// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PosterList /> snapshots should match snapshot 1`] = `
<div
  className="sc-bdVaJa lnRSUj"
>
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/gpxjoE0yvRwIhFEJgNArtKtaN7S.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/4RnCeRzvI1xk5tuNWjpDKzSnJDk.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/3PEAkZHa8ehfUkuKbzmQNRTTAAs.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/zDJ4GVch7Iw2i0j84ceoElRUZUR.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/gpxjoE0yvRwIhFEJgNArtKtaN7S.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/4RnCeRzvI1xk5tuNWjpDKzSnJDk.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/3PEAkZHa8ehfUkuKbzmQNRTTAAs.jpg"
  />
  <img
    alt="This will be dynamic and will be used for accessibility"
    src="https://image.tmdb.org/t/p/w185/zDJ4GVch7Iw2i0j84ceoElRUZUR.jpg"
  />
</div>
`;
